### ISSUE TITLE
False positive of `no-nonstandard-iterator-prototype-properties`

### ISSUE THREAD

**Issue by zloirock** [eslint-community/eslint-plugin-es-x#228]
```js
[].keys().next() // => Non-standard 'Iterator.prototype.next' property is forbidden
```
